pkgs/applications/office/pdfrankenstein/default.nix

```diff
@@ -0,0 +1,45 @@
+{ buildGo118Module, fetchFromGitHub, lib, pkg-config, makeWrapper, gdk-pixbuf, inkscape, poppler_utils, qpdf, gtk3, glib }:
+
+buildGo118Module rec {
```

Comment:
It is better to use buildGomodule and then overwrite the version in pkgs/top-level/all-packages.nix.
That way when the default version gets updated (to 118, or further) it's more easily replaced